egov/egov-collection/src/main/java/org/egov/collection/utils/CollectionsUtil.java

```diff
@@ -764,6 +764,7 @@ public ReceiptAmountInfo updateReceiptDetailsAndGetReceiptAmountInfo(BillReceipt
         RestTemplate restTemplate = new RestTemplate();
         HttpHeaders headers = new HttpHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
+        billReceipt.setTenantId(ApplicationThreadLocals.getCityName());
         String json = null;
```

Comment:
ApplicationThreadLocals.getCityName() won't be tenant ID in ERP use TenantId instad